setup.cfg

```diff
@@ -1,12 +0,0 @@
-[isort]
```

Comment:
Why are we removing `setup.cfg`?

Reply:
Not a great reason, but here's what I had in mind: when we move to be a service, we won't need `setup.py` (correct me if I'm wrong). I know `setup.cfg` is not necessarily linked to it, but I felt it'd be a good idea to have all this stuff in 1 central configuration file, namely `tox.ini` since that's where we already store stuff, rather than spread out all over the place almost arbitrarily. Again this change isn't necessarily called for in this PR, but I made it a "drive-by" change while changing other stuff in the root folder.